time stamp

# flag if human is looking around ( O4 ) 
bool humanLookingAround

# details about the gesture, like gaze directions and angles 
string headGestureDetails 

